config/sidekiq.yml

```diff
@@ -58,3 +58,7 @@
       interval: 1 minute
       class: Scheduler::SuspendedUserCleanupScheduler
       queue: scheduler
+    software_update_check_scheduler:
+      interval: 30 minutes
```

Comment:
Might be good to make this configurable via environment variable?

also, is every 30 mins

Reply:
I wonder if we shouldn't figure out how to have dotenv import `.env.production` before `config/sidekiq.yml` is parsed, though, this would come in handy in another PR as well.